commit 6e04e13b5df66fee39667611ffa5cc7c93957b7c (refs/changes/93/2993/1)
Author: Pedro Palazon Candel <pedro@joyent.com>
Date:   2017-11-23T17:59:46+01:00 (1 year, 10 months ago)
    
    HEAD-2377 SAPI zone should use its own admin IP address for sapi-url, not sapi's DNS domain

diff --git a/scripts/headnode.sh b/scripts/headnode.sh
index 7e601941..ce58f72b 100755
--- a/scripts/headnode.sh
+++ b/scripts/headnode.sh
@@ -638,7 +638,6 @@ upload_values
 download_metadata
 write_initial_config
 registrar_setup
-mdata-put sapi-url http://${CONFIG_sapi_domain}
 HERE
         setup_state_add "sapi_bootstrapped"
     fi
